clearwater-snmpd/etc/snmp/snmpd.conf.clearwater-snmpd

```diff
@@ -216,6 +216,10 @@ view clearwater included .1.3.6.1.2.1.121.1.1.1
 # MIBs defined by Project Clearwater
 view clearwater included .1.2.826.0.1.1578918
 
+# Create an internal community that can access anything but is only usable locally.
+rocommunity clearwater-internal 127.0.0.1/32 -V clearwater-internal
```

Comment:
`127.0.0.0/8` if you want the whole loop-back space (as @rkd-msw) pointed out to me the other day.